src/app/portlets/shared/dot-content-types-edit/dot-content-types-edit.component.html

```diff
@@ -8,6 +8,7 @@
         *ngIf="data.id"
         [layout]="layout"
         [loading]="loadingFields"
+        [contentType]="data"
```

Comment:
test?

Reply:
on `dot-content-types-edit.component.spec.ts` to check that `dot-content-type-fields-drop-zone` has all those inputs